src/main/scala/zio/rocksdb/service/WriteBatch.scala

```diff
@@ -0,0 +1,10 @@
+package zio.rocksdb.service
+
+import org.{ rocksdb => jrocks }
+import zio.Task
+
+trait WriteBatch {
+  def put(key: Array[Byte], value: Array[Byte]): Task[Unit]
+
+  def put(cfHandle: jrocks.ColumnFamilyHandle, key: Array[Byte], value: Array[Byte]): Task[Unit]
+}
```

Comment:
I don't see the value of representing batches as services. A thin wrapper with proper set of constructors and finalizers would be sufficient, imho. Also, having in mind the resource safety constraints you mentioned, I think it's safe to say that https://github.com/zio/zio-rocksdb/pull/29 is not sufficient.

@iravid WDYT?